Description: fix breakage caused by QtQml "null" assignment semantics change in 5.8
Origin: upstream, https://code.qt.io/cgit/qt/qtpim.git/commit/?id=70733640ccdd61b4
Last-Update: 2017-07-07

--- a/tests/auto/versit/qversitcontactimporter/tst_qversitcontactimporter.cpp
+++ b/tests/auto/versit/qversitcontactimporter/tst_qversitcontactimporter.cpp
@@ -1580,7 +1580,11 @@
     {
         QTest::newRow("null denoting an empty variant")
                 << QString("name")
+#if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
+                << QVariant(static_cast<QVariant::Type>(QMetaType::Nullptr))
+#else
                 << QVariant()
+#endif
                 << jsonArrayWith.arg("null")
                 << true;
         QTest::newRow("compact json with extra whitespace removed")
--- a/tests/auto/versitorganizer/qversitorganizerimporter/tst_qversitorganizerimporter.cpp
+++ b/tests/auto/versitorganizer/qversitorganizerimporter/tst_qversitorganizerimporter.cpp
@@ -44,6 +44,10 @@
 #include "tst_qversitorganizerimporter.h"
 #include <QtTest/QtTest>
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
+#include <cstddef>
+#endif
+
 QTORGANIZER_USE_NAMESPACE
 QTVERSITORGANIZER_USE_NAMESPACE
 
@@ -1805,7 +1809,11 @@
     {
         QTest::newRow("null denoting an empty variant")
                 << QString("name")
+#if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
+                << QVariant::fromValue<std::nullptr_t>(nullptr)
+#else
                 << QVariant()
+#endif
                 << jsonArrayWith.arg("null")
                 << true;
         QTest::newRow("compact json with extra whitespace removed")
